ooni/nettests/blocking/web_connectivity.py

```diff
@@ -77,6 +77,12 @@ class WebConnectivityTest(httpt.HTTPTest, dnst.DNSTest):
     requiresTor = False
     followRedirects = True
 
+    # These are the options to be shown on the GUI
+    simpleOptions = [
+        {"name": "url", "type": "text"},
```

Comment:
I see from below that the `file/url` type exists. So, why isn't a `url` field of type `file/url`?